### ISSUE TITLE
Training projects: Add skeleton docs

### ISSUE THREAD

**Issue by stephlocke** [lockedata/pRojects#1]
Create some templates that can be added to createTrainingProject

**Comment by stephlocke** [comment]
Key Q: what package(s) to support for handouts?
Key Q: what package(s) to support for slides?

**Comment by jonmcalder** [comment]
So I assume we can aim to support a few (or a few combinations) and incorporate these into the templates referred to in #14?

Maybe start with Rmarkdown options like like [ioslides, slidy, reveal etc](http://rmarkdown.rstudio.com/lesson-11.html) and then extend with more 3rd party options, custom css etc from there?

**Comment by stephlocke** [comment]
Good shout on the slides options. 

For handouts I'm thinking [bookdown](https://github.com/rstudio/bookdown) and [tufte](https://github.com/rstudio/tufte), plus standard html_documents, pdf etc.

**Comment by stephlocke** [comment]
We've now got bookdown and tufte in the training projects for handouts and revealjs for the slides so far. (over and above rmarkdown)

**Comment by jonmcalder** [comment]
Based on the feedback from my [Twitter poll](https://twitter.com/jonmcalder/status/862037896227942401):

`xaringan` and `revealjs` seem to be the most popular, and should be adequately covered (for now) by the addition of #25 and #27.

`ioslides`, `slidy` and `beamer` all also got some votes, but since these don't have additional package dependencies and R Markdown templates for each of these are already distributed with RStudio I don't think any additional/explicit provision for these is necessary? (I did add a brief note about these in the vignette though)

Since "other" had the majority of the votes, I'm hoping most of these were for `xaringan` (and possibly some for `beamer`), but I'm not really sure.

The only others that I saw explicitly mentioned in the Twitter discussion were [beamer metropolis](https://github.com/eddelbuettel/samples-rmarkdown-metropolis) and [ioslides plus](https://github.com/koncina/iosp), but these seam like fairly niche additions.
